File: src/test/Ruby/iOS.rb
require 'rubygems'
require 'appium_lib'



desired_caps = {

    'deviceName': 'iPhone',
    'platformVersion': '13.2.2',
    'udid': 'e3a2211901a618a832d92b5cecec27ca049f0a0d',
    'platformName': 'iOS'

}
appium_driver = Appium::Driver.new({
    'caps' => desired_caps,
    'appium_lib' => {
        :server_url => "http://20.204.77.152:4444"
    }}, true)
driver = appium_driver.start_driver
wait = Selenium::WebDriver::Wait.new(:timeout => 30)

driver.save_screenshot "iOSRuby.png"


driver.quit

File: src/test/Python/iOS.py
import self as self
from appium import webdriver
import unittest

desired_caps = dict(
    platformVersion='13.2.2',
    deviceName='iPhone',
    platformName='iOS',
    udid='e3a2211901a618a832d92b5cecec27ca049f0a0d'
)

self.driver = webdriver.Remote('http://20.204.77.152:4444', desired_caps)

print(self.driver.page_source)
self.driver.save_screenshot('iOSPython.png')

self.driver.quit()
